src/terminal/init.lua

```diff
@@ -1510,7 +1237,7 @@ M.box_fmt = setmetatable({
 -- @tparam[opt=false] boolean lastcolumn whether to draw the last column of the terminal
 -- @treturn string ansi sequence to write to the terminal
 -- @within lines
-function M.boxs(height, width, format, clear, title, lastcolumn)
+function M.boxs(height, width, format, clear_flag, title, lastcolumn)
```

Comment:
you renamed the parameter, which means you must also update the doc-comment above. Running ldoc will probably give you an error now due to the name mismatch.